spectator/config.h

```diff
@@ -12,17 +13,17 @@ class Config {
   Config() = default;
   Config(const Config&) = default;
 
-  Config(std::map<std::string, std::string> c_tags, int read_to, int connect_to,
+  Config(std::map<std::string, std::string> c_tags, int read_to_ms, int connect_to_ms,
          int batch_sz, int freq, std::string publish_uri)
       : common_tags{std::move(c_tags)},
-        read_timeout{read_to},
-        connect_timeout{connect_to},
+        read_timeout{std::chrono::milliseconds{read_to_ms}},
+        connect_timeout{std::chrono::milliseconds{connect_to_ms}},
         batch_size{batch_sz},
         frequency{freq},
         uri{std::move(publish_uri)} {}
   std::map<std::string, std::string> common_tags;
-  int read_timeout;     // in seconds
-  int connect_timeout;  // in seconds
+  std::chrono::milliseconds read_timeout;
+  std::chrono::milliseconds connect_timeout;
   int batch_size;
   int frequency;  // in seconds
```

Comment:
Would probably be best to have same units throughout in the ctor, a bit confusing the frequency is left in seconds.

Reply:
I'll update frequency in the next release to take milliseconds as well. Thanks for the feedback.